packages/ckeditor5-font/src/ui/colorui.ts

```diff
@@ -139,10 +139,18 @@ export default class ColorUI extends Plugin {
 				editor.editing.view.focus();
 			} );
 
+			dropdownView.on( 'colorChange', ( evt, data ) => {
+				editor.execute( this.commandName, data );
+			} );
+
 			dropdownView.on( 'change:isOpen', ( evt, name, isVisible ) => {
 				// Grids rendering is deferred (#6192).
 				dropdownView.colorTableView!.appendGrids();
 
+				if ( editor.config.get( this.componentName )!.colorPicker !== false ) {
+					dropdownView.colorTableView!.appendColorPicker();
```

Comment:
This should be internally done by the `ColorTableView` class, in the constructor.

Since the constructor is [already getting plenty of options](https://github.com/ckeditor/ckeditor5/blob/87f5544a878511d4fabc7faf9108a4b3c0a07472/packages/ckeditor5-font/src/ui/colortableview.ts#L147), you should pass color picker configuration the same way.